build/release/upload_distribution.sh

```diff
@@ -47,35 +46,25 @@ if [[ ! -d "$this_script_directory" ]]; then
 fi
 
 readonly optaplanner_project_root=$this_script_directory/../..
-readonly optaplanner_distribution=$optaplanner_project_root/optaplanner-distribution
-readonly optaplanner_distribution_build_dir="$optaplanner_distribution/target/optaplanner-distribution-$version"
+readonly optaplanner_docs_build_dir="$optaplanner_project_root/optaplanner-docs/target"
 
-if [[ ! -d "$optaplanner_distribution_build_dir" ]]; then
-  echo "The OptaPlanner distribution does not exist. Please run the build with the full profile enabled."
+if [[ ! -d "$optaplanner_docs_build_dir" ]]; then
+  echo "The $optaplanner_docs_build_dir directory does not exist. Please run the build with the full profile enabled."
   exit 1
 fi
 
 # Create the directory structure .../release/${version}
 readonly temp_release_directory=/tmp/optaplanner-release-$version
-readonly local_optaplanner_downloads=$temp_release_directory/downloads/release
 readonly local_optaplanner_docs=$temp_release_directory/docs/release
 if [ -d "$temp_release_directory" ]; then
   rm -Rf "$temp_release_directory";
 fi
 mkdir -p "$local_optaplanner_docs/$version/optaplanner-docs"
 mkdir -p "$local_optaplanner_docs/$version/optaplanner-javadoc"
-mkdir -p "$local_optaplanner_downloads/$version"
-
-# Upload the OptaPlanner distribution.zip.
-cp "$optaplanner_distribution/target/optaplanner-distribution-$version.zip" "$local_optaplanner_downloads/$version"
-
-readonly remote_shell="ssh -oKexAlgorithms=+diffie-hellman-group1-sha1 -i $optaplanner_ssh_key"
-create_latest_symlinks "$local_optaplanner_downloads" "$version"
-rsync -a -r -e "$remote_shell" --protocol=28 "$local_optaplanner_downloads/.." "$remote_optaplanner_downloads"
 
 # Upload the documentation and Javadoc.
-cp -r "$optaplanner_distribution_build_dir"/reference_manual/* "$local_optaplanner_docs/$version/optaplanner-docs"
-cp -r "$optaplanner_distribution_build_dir"/javadocs/* "$local_optaplanner_docs/$version/optaplanner-javadoc"
+cp -r "$optaplanner_docs_build_dir"/generated-docs/* "$local_optaplanner_docs/$version/optaplanner-docs"
+cp -r "$optaplanner_docs_build_dir"/aggregated-javadocs/apidocs/* "$local_optaplanner_docs/$version/optaplanner-javadoc"
```

Comment:
The assembly of optaplanner-distribution used to move these artifacts from their original location to `reference-manual` and `javadocs`. Now we access them directly.